### PR TITLE
feat: add novu

### LINKED ISSUE #662: App Suggestion: Novu, the open-source notification infrastructure
# The open-source notification infrastructure for developers.

Simple components and APIs for managing all communication channels in one place: Email, SMS, Direct, and Push.

## Introduction
Novu is an open-source notification infrastructure built for the engineering teams to help them build rich product notification experiences without constantly re-inventing the wheel.

## Why?
Building a notification layer is that thing that we as developers have to do for every new project. It usually starts the same, sending just a simple email, and after a while, you'll find yourself managing hundreds of notifications across multiple channels. Novu’s goal is to help developers create meaningful transactional communication between the product and its users of it. All with an easy-to-use API and outstanding developer experience.

Website: https://novu.co/
Deploy using Docker: https://docs.novu.co/docs/overview/docker-deploy

image

### PR DESCRIPTION
**Pull request by ronaldloyko**
This pull request adds the one-click-app for [Novu](https://novu.co).

## Checks

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

## References

- closes #662

### CODE REVIEW

File: public/v4/apps/novu.yml [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+        end: |-
+            Novu has been successfully deployed! It might take few moments before it's fully started.
+            You can access it at `http://$$cap_appname.$$cap_root_domain`.
+    variables:
+        - id: $$cap_NOVU_IMAGE_VERSION
+          label: Application | Version
+          description: Version tag of Novu's Docker images. Check out their valid tags at https://github.com/novuhq/novu/pkgs/container/novu%2Fapi
+          defaultValue: '0.8.0'
+          validRegex: /.{1,}/
+        - id: $$cap_NOVU_IMAGE_URL
```
Comment (githubsaturn) [inline public/v4/apps/novu.yml:110]: nit:
This seems like an overkill for a variable. It's best to keep variable list short to the ones that are typically changed for the end users. The image URL is most likely going to be fixed and not changed.

Comment (ronaldloyko) [reply public/v4/apps/novu.yml:110]: Allright, I'll keep that in mind.
Thanks.

**Comment by joingardens** [comment]
@ronaldloyko did you use Localstack with your novu instance? Couldn't get it to work with Caprover

**Comment by ronaldloyko** [comment]
@joingardens I don't use Novu at the moment, but it used to work when I used it on CapRover.
Do you have any error logs?

**Pull request merged**